app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ReviewsController < ApplicationController
+  def create
+    @landmark = Landmark.find(params[:landmark_id])
+    @review = Review.new(review_params)
+    @review.landmark = @landmark
```

Comment:
I'm not sure of the best way to do that as the `@landmark` is used to both populate the `landmark_id` in `@review` and also to redirect to the landmark show page. Unless you mean a private method that is dedicated to that set of actions and returns `@landmark`